src/openforms/appointments/contrib/jcc/plugin.py

```diff
@@ -340,10 +356,13 @@ def get_appointment_details(self, identifier: str) -> AppointmentDetails:
             qrcode = client.service.GetAppointmentQRCodeText(appID=identifier)
 
             app_products = []
-            for pid in details.productID.split(","):
-                product = client.service.getGovProductDetails(productID=pid)
-
-                app_products.append(Product(identifier=pid, name=product.description))
+            product_ids = Counter(details.productID.split(","))
+            for product_id, count in product_ids.items():
+                _product = client.service.getGovProductDetails(productID=product_id)
+                product = Product(
+                    identifier=product_id, name=_product.description, amount=count
```

Comment:
I see in the WSDL in the mocks, that the description can appear also "zero" times. Would this cause an error in this case?

Reply:
I'll try it!

Reply:
Added and fallback to empty string, it returned `None` otherwise (which doesn't fit the type definitions)